src/conv/k-g/k_parser.cpp

```diff
@@ -477,6 +759,89 @@ bool parse_k
 			}
 			break;
 		    }
+		    case KState::Element_Beam: {
+			if (elementBeamOptions.size() == 0 || elementOptionsCounter == 0) {
+			    if (tokens.size() < 10) {
+				std::cout << "Too short ELEMENT_BEAM in k-file" << fileName << "\n";
+				break;
+			    }
+			    int eid = stoi(tokens[0]);
+
+			    if (data.elements.find(eid) != data.elements.end()) {
+				std::cout << "Duplicat Element ID" << eid << " in k-file" << fileName << "\n";
+				break;
+			    }
+
+			    KElement element;
+
+			    for (int i_n = 0; i_n < 5; ++i_n) {
+				element.nodes.push_back(stoi(tokens[i_n + FirstNode]));
+			    }
+
+			    data.elements[eid] = element;
+
+			    int pid = stoi(tokens[1]);
+			    data.parts[pid].elements.insert(eid);
+			    break;
+			}
+			else if ((elementBeamOptions.size()) > 0 && (elementOptionsCounter < elementBeamOptions.size())) {

```

Comment:
Hmm, if `elementBeamOptions.size()` is greater than `elementOptionsCounter`, then it must be greater than 0 too.

Reply:
you are right, I will fix it